grafana/resource_dashboard_test.go

```diff
@@ -15,58 +16,74 @@ func TestAccDashboard_basic(t *testing.T) {
 
 	var dashboard gapi.Dashboard
 
-	resource.Test(t, resource.TestCase{
-		PreCheck:          func() { testAccPreCheck(t) },
-		ProviderFactories: testAccProviderFactories,
-		CheckDestroy:      testAccDashboardCheckDestroy(&dashboard),
-		Steps: []resource.TestStep{
-			{
-				// Test resource creation.
-				Config: testAccExample(t, "resources/grafana_dashboard/_acc_basic.tf"),
-				Check: resource.ComposeTestCheckFunc(
-					testAccDashboardCheckExists("grafana_dashboard.test", &dashboard),
-					resource.TestCheckResourceAttr("grafana_dashboard.test", "id", "basic"),
-					resource.TestCheckResourceAttr("grafana_dashboard.test", "uid", "basic"),
-					resource.TestCheckResourceAttr(
-						"grafana_dashboard.test", "config_json", `{"title":"Terraform Acceptance Test","uid":"basic"}`,
-					),
-				),
-			},
-			{
-				// Updates title.
-				Config: testAccExample(t, "resources/grafana_dashboard/_acc_basic_update.tf"),
-				Check: resource.ComposeTestCheckFunc(
-					testAccDashboardCheckExists("grafana_dashboard.test", &dashboard),
-					resource.TestCheckResourceAttr("grafana_dashboard.test", "id", "basic"),
-					resource.TestCheckResourceAttr("grafana_dashboard.test", "uid", "basic"),
-					resource.TestCheckResourceAttr(
-						"grafana_dashboard.test", "config_json", `{"title":"Updated Title","uid":"basic"}`,
-					),
-				),
-			},
-			{
-				// Updates uid.
-				// uid is removed from `config_json` before writing it to state so it's
-				// important to ensure changing it triggers an update of `config_json`.
-				Config: testAccExample(t, "resources/grafana_dashboard/_acc_basic_update_uid.tf"),
-				Check: resource.ComposeTestCheckFunc(
-					testAccDashboardCheckExists("grafana_dashboard.test", &dashboard),
-					resource.TestCheckResourceAttr("grafana_dashboard.test", "id", "basic-update"),
-					resource.TestCheckResourceAttr("grafana_dashboard.test", "uid", "basic-update"),
-					resource.TestCheckResourceAttr(
-						"grafana_dashboard.test", "config_json", `{"title":"Updated Title","uid":"basic-update"}`,
-					),
-				),
-			},
-			{
-				// Importing matches the state of the previous step.
-				ResourceName:            "grafana_dashboard.test",
-				ImportState:             true,
-				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"message"},
-			},
-		},
-	})
+	for _, useSHA256 := range []bool{false, true} {
```

Comment:
Review without whitespace changes. I reused the basic test but when it's a sha256, the expected state is different

Reply:
It wasn't a comment for him. Just a general "if you review this test, review it without whitespace". I wrote that test 😄

Reply:
Aaaahhh ok, my bad, then.

So, just to see if I got this straight: the test fails?

Reply:
No, the test passes. It just shows lots of changes because indentation changes